### ISSUE TITLE
It's possible validate a chained select?

### ISSUE THREAD

**Issue by guille1194** [formvalidation/support#680]
I have an error when i try to validate a chained select, because the select put the check when is empty.

Here is the code without the validator: http://codepen.io/CSWApps/pen/Bemst

**Comment by guille1194** [comment]
someone understands what I try to do?

**Comment by guille1194** [comment]
Please, i need an answer

**Comment by guille1194** [comment]
please i need an answer or suggestion or idea.
Thanks.

**Comment by guille1194** [comment]
Sorry, but I really need your help.

**Comment by guille1194** [comment]
ok, I solved the logic of the problem I had, now I just have to apply the formvalidation, so, that is my question.

Here is the code: http://codepen.io/guille1194/pen/grJNzg

Please help me, I'm doing everything possible.

**Comment by guille1194** [comment]
ok, i only want to know if it's possible to make the code: [http://codepen.io/guille1194/pen/grJNzg](code) with a formvalidation, or i put the method nextAll() in the selects with formvalidation.

I really need an answer, thank you.

**Comment by guille1194** [comment]
Please, i need an answer, i'm still waiting for 2 days, and i almost give all the solution.

Thanks.

**Comment by guille1194** [comment]
ok, here is the code in a pen, and i don't know what is wrong.

http://codepen.io/guille1194/pen/grJNzg

**Comment by nghuuphuoc** [comment]
I am trying to fix it for me. Do you want that all `color`, `size` and `qty` to be required?

**Comment by guille1194** [comment]
no, what I try to do is that when the select is disabled , mark it as error with formvalidation. Thanks for reply.

**Comment by nghuuphuoc** [comment]
How we can mark the field as error if it doesn't have any validations rules (such as required)?

**Comment by guille1194** [comment]
In fact, how  i do it?

**Comment by guille1194** [comment]
Yes, In fact, i need the required, but how can i solve it?

**Comment by guille1194** [comment]
Can you upload the code, please?

**Comment by nghuuphuoc** [comment]
Here is the [working jsfiddle](http://jsfiddle.net/nghuuphuoc/bwysr7pc/3/) which requires all `color`, `size`, `qty` fields.

Please notice that:
1) By default, the disabled input will be ignored by the plugin. Here we have 3 disabled select elements initially, so in order to set the validator rules for them, we need to use the [excluded](http://formvalidation.io/settings/#form-excluded) option.

2) Whenever you change the value of field dynamically, you need to revalidate it by using [revalidateField](http://formvalidation.io/api/#revalidate-field) method.

These two principles are very important and are mentioned at the [compatibility](http://formvalidation.io/examples/#compatibility) section (So, before asking,  please read the documentations :).

![select](https://cloud.githubusercontent.com/assets/4226036/17956645/9abc73d2-6ab5-11e6-9beb-b868abcadef6.gif)

**Comment by guille1194** [comment]
Wow, thanks and sorry for the presure.

:+1:

**Comment by nghuuphuoc** [comment]
No worries. Sometime I can't respond in time (sorry about that), but feel free to ask 👍

**Comment by guille1194** [comment]
Ok, you don't know how to make the filling of the first selection, others are marked as incorrect, thanks.
